tagbot/action/repo.py

```diff
@@ -590,6 +590,7 @@ def create_release(self, version: str, sha: str) -> None:
     def handle_error(self, e: Exception) -> None:
         """Handle an unexpected error."""
         allowed = False
+        internal = False
```

Comment:
sorry if I missed it, but does `internal` ever get set to `True`?

Reply:
I think this is actually supposed to default to True